src/backend_telemetry_buffer.rs

```diff
@@ -167,7 +173,7 @@ mod _prom_ext {
     }
 
     /// Adds a `backend_telemetry_rec` record to a backend-local ring buffer.
-    #[pg_extern(volatile, strict, create_or_replace, requires = ["backend_telemetry_rec_decl"])]
+    #[pg_extern(volatile, strict, create_or_replace, sql = false)]
```

Comment:
Sorry, my feedback in the previous review may have been confusing, I didn't mean that we should set `sql = false` here.

Reply:
It doesn't work otherwise. Generated DDL references `backend_telemetry_rec` that is now defined at a later point.

Reply:
Of course! Sorry I missed that. Hmm... That's somehow quite annoying...